inc/deploygroup.class.php

```diff
@@ -114,7 +114,7 @@ public function defineTabs($options = [])
         $this->addDefaultFormTab($ong);
 
         $count = self::getMatchingItemsCount("PluginGlpiinventoryTaskjob");
-        $ong[$this->getType() . '$task'] = self::createTabEntry(_n('Associated task', 'Associated tasks', $count), $count);
+        $ong[$this->getType() . '$task'] = self::createTabEntry(__('Associated tasks', 'glpiinventory'), $count);
```

Comment:
I do not agree with this change. singular/plural form exists elsewhere, it's better to keep only one occurrence for translation.